lib/publiccloud/utils.pm

```diff
@@ -116,6 +116,7 @@ sub registercloudguest {
     # Check what version of registercloudguest binary we use
     $instance->ssh_script_run(cmd => "rpm -qa cloud-regionsrv-client");
     $instance->ssh_script_retry(cmd => "sudo $suseconnect -r $regcode", timeout => 420, retry => 3, delay => 120);
+    assert_script_run('ssh -O exit ' . $instance->username . '@' . $instance->public_ip);
```

Comment:
I think we need to do this on every console that is opened.

Reply:
I think your suggestion applies only to `TUNNELED=1` tests. But even in those we don't (re)register the system after `publiccloud/ssh_interactive_start` test module.

The `publiccloud::utils::registercloudguest` subroutine is currently used in the following test modules:
- `publiccloud/registration`
- `publiccloud/migration`
- `publiccloud/slem_basic`
- `publiccloud/run_ltp`
- `publiccloud/check_registercloudguest`
- `publiccloud/storage_perf`
- `sles4sap/publiccloud/qesap_terraform`
- `sles4sap/cloud_netconfig/deploy`

I think think that we are safe. But please feel free to suggest more VRs.

Reply:
No you're right. Let's proceed.